test/functionalTest/cases/1661_bad_input_on_notification_error/bad_input_on_notification_error_with_repeated_alarms_api_v1.test

```diff
@@ -0,0 +1,309 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
..._repeated_alarms....test -> ..._reloged_alarms....test (closer to the meaning of the -relogAlarms CLI param)